src/components/LineGutter/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { object } from "prop-types";
+
+import React, { Component } from "react";
+
+import { lineGutter } from "./index.module.css";
+
+/**
+ * The line number of a single line.
+ * The anchor contained within is interactive, and will highlight the
+ * entire line upon selection.
+ */
```

Comment:
Is this for `LineNumber`

Reply:
Well, I did copy paste this.. forgot to update this 🙃